app/javascript/dashboard/routes/dashboard/settings/reports/components/SLA/SLATable.vue

```diff
@@ -50,7 +50,7 @@
       :current-page="currentPage"
       :total-count="totalCount"
       :page-size="pageSize"
-      class="bg-slate-25 dark:bg-slate-900 sticky bottom-0 border-none mt-4"
+      class="border-none"
```

Comment:
@muhsin-k Do we need `border-none` here? In the `table-footer` component, there is no border.
```suggestion
```
```
<footer
    v-if="isFooterVisible"	  
    class="h-12 flex items-center justify-between px-6"
/>	 
```